pom.xml

```diff
@@ -921,6 +921,7 @@
               <java.io.tmpdir>${project.build.directory}/tmp</java.io.tmpdir>
               <project.version>${project.version}</project.version>
               <jacoco-agent.destfile>target/jacoco.exec</jacoco-agent.destfile>
+              <test.mode>true</test.mode>
```

Comment:
Why do we need this?

Reply:
You can see the 2nd point listed above

Reply:
Since `RssUtils#getHostIp` is a method in a utility class and does not accept any input parameters, we can only use environment variables to inform the internal logic that the unit test is being executed. This approach requires the least amount of changes and is the most reasonable.

Reply:
Em... it seems not enough reasonable. If we should support the similar scenary, we should limit to use it in the test mode.

Reply:
When running unit tests, they are executed on a single machine and do not interact with other machines. Therefore, unit tests should not require a valid broadcast IP address. Currently, we do not support scenarios where the broadcast IP address of the host machine is 0.0.0.0. I think this is an unreasonable limitation.

Reply:
You should add the comment, too.

Reply:
Added.